packages/projects-docs/pages/learn/plans/workspace.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: Teams
+authors: ['CompuIves']
+description:
+  A team makes it easier to collaborate on code with others. In a team you share
+  ownership of repositories, sandboxes and templates.
+---
+
+import { Callout } from 'nextra-theme-docs'
+
+# Workspaces
+![Workspaces](../images/workspace-selector.png)
+
+CodeSandbox uses **Workspaces** to provide personal and collaborative experiences. You can think of workspaces as the boundaries for collaboration on CodeSandbox: use a [personal workspace](#personal-workspace) for public, individual development work, and use a [Pro workspace](#pro-workspace) for any type of work requiring privacy or collaboration with others.
```

Comment:
personal `or` collaborative?

Reply:
hmm, I think this whole section needs more revision since it reinforces that personal is for individual work and pro is for collab